src/memefactory/ui/dank_registry/submit_page.cljs

```diff
@@ -98,15 +98,17 @@
                             :label "Upload a file"
                             :comment "Upload image with ratio 2:3 and size less than 1.5MB"
                             :file-accept-pred (fn [{:keys [name type size] :as props}]
-                                                (js/console.log "Veryfing acceptance of file of type : " type " and size : " size)
-                                                (and (#{"image/png" "image/gif" "image/jpeg"} type)
+                                                (log/debug "Veryfing acceptance of file" {:name name :type type :size size})
+                                                ;; TODO : mp4s
```

Comment:
What else is there to do with mp4s? since "video/mp4" is already added among types?

Reply:
gifv will need a couple more changes (inputs library, tiles), addressed in a separate PR. This one is about svg, removing